app/src/cobrand/java/cm/aptoide/pt/account/LoginPreferences.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017.
+ * Modified by Marcelo Benites on 10/02/2017.
+ */
+
+package cm.aptoide.pt.account;
+
+import android.content.Context;
+import com.google.android.gms.common.GoogleApiAvailability;
+
+public class LoginPreferences {
+
+  private final Context context;
```

Comment:
These two variables are not being used. If you moved the provides method to the respective FlavourApplicationModule, then you could have an instance in cobrand where the constructor would be empty. In vanilla flavour application module the constructor would be the same, in cobrand it would be an empty one since it doesnt have any dependencies.